CHANGELOG.md

```diff
@@ -44,6 +44,7 @@
 - Route spans according to trace_id. ([#3387](https://github.com/getsentry/relay/pull/3387))
 - Log span when encountering a validation error. ([#3401](https://github.com/getsentry/relay/pull/3401))
 - Optionally skip normalization. ([#3377](https://github.com/getsentry/relay/pull/3377))
+- Remove `ShardedProducer`. ([#3415](https://github.com/getsentry/relay/pull/3415))
```

Comment:
Can we mark this as a breaking change and put it under the other breaking change in `**Features**`?

Reply:
@iambriccardo can you also mark it as a breaking change?

Reply:
How do we mark it as a breaking change?

Reply:
Nvm just saw

Reply:
And now move it below the other breaking change :p.

Yeah it's not very fancy, we should probably at some point rework our changelog a bit.

Reply:
I feel like we should just create a set of rules that one should follow since right now it's a bit copy pasty.